### DISCUSSION TITLE
请问这个是什么问题，tidevice运行正常

### DISCUSSION THREAD

**Discussion by 373424129** [SonicCloudOrg/sonic-server#420] [Q&A] [answered]
![1707038586528_F8A5E5D9-0417-4e71-8B81-BA9FB49010D8](https://github.com/SonicCloudOrg/sonic-server/assets/40819159/92f91910-aed7-4af2-b187-14b8723e2dc7)
![1707038586528_F8A5E5D9-0417-4e71-8B81-BA9FB49010D8](https://github.com/SonicCloudOrg/sonic-server/assets/40819159/0237a76e-92ac-4f79-a75b-4f90c7c46d28)

**Comment by jongyu** [comment]
我也遇到这个问题，go-ios、tidevice都可以正常启动wda，但是sib却不行

**Reply by ZhouYixun** [reply]
暂时解决方案是用xcode13构建wda，我们还在找原因

**Reply by ZhouYixun** [reply]
https://github.com/SonicCloudOrg/sonic-ios-bridge/releases/tag/v1.3.18

v1.3.18已修复

**Comment by ZhouYixun** [comment, marked as answer]
https://github.com/SonicCloudOrg/sonic-ios-bridge/releases/tag/v1.3.18

v1.3.18已修复

**Comment by 373424129** [comment]
感谢🙏，已验证，没有问题了。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Eason ***@***.***&gt;
发送时间: 2024年2月27日 10:28
收件人: SonicCloudOrg/sonic-server ***@***.***&gt;
抄送: burden ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [SonicCloudOrg/sonic-server] 请问这个是什么问题，tidevice运行正常 (Discussion #420)





 
https://github.com/SonicCloudOrg/sonic-ios-bridge/releases/tag/v1.3.18
 
v1.3.18已修复

**Comment by 373424129** [comment]
用xcode15 构建的webdriver 运行 monkey 测试的时候没有测试完成强行退出，会导致无法再次启动 webdriveragent



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Eason ***@***.***&gt;
发送时间: 2024年2月27日 10:29
收件人: SonicCloudOrg/sonic-server ***@***.***&gt;
抄送: burden ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [SonicCloudOrg/sonic-server] 请问这个是什么问题，tidevice运行正常 (Discussion #420)





 
https://github.com/SonicCloudOrg/sonic-ios-bridge/releases/tag/v1.3.18
 
v1.3.18已修复

**Reply by ZhouYixun** [reply]
这似乎不是sonic的问题？

**Comment by 373424129** [comment]
更新 sonic 最新代码后这个问题没有了👌。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Eason ***@***.***&gt;
发送时间: 2024年2月27日 18:25
收件人: SonicCloudOrg/sonic-server ***@***.***&gt;
抄送: burden ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [SonicCloudOrg/sonic-server] 请问这个是什么问题，tidevice运行正常 (Discussion #420)





 
这似乎不是sonic的问题？
